OpenSim/Simulation/Model/ForceSet.cpp

```diff
@@ -313,11 +209,11 @@ Set<Actuator>& ForceSet::updActuators()
  */
 void ForceSet::updateActuators()
 {
+    _actuators.setMemoryOwner(false);
     _actuators.setSize(0);
-    for (int i = 0; i < getSize(); ++i)
-    {
+    for (int i = 0; i < getSize(); ++i) {
         Actuator* act = dynamic_cast<Actuator*>(&get(i));
-        if (act != NULL)  _actuators.adoptAndAppend(act);
+        if (act)  _actuators.adoptAndAppend(act);
```

Comment:
I think some compilers don't like using pointer as a boolean?

Reply:
Wouldn't it cause a compiler warning if that if that were true? In that case we have to do a sweep of the entire codebase, because I think there are several other instances.

Reply:
We do this extensively throughout the code base.